ekip/nationalparks/models.py

```diff
@@ -41,7 +41,7 @@ class FieldTripSite(models.Model):
     phone = PhoneNumberField(null=True)
     phone_extension = models.IntegerField(null=True)
     city = models.CharField(max_length=128, null=True)
-    state = USStateField(blank=False, null=False)
+    state = USStateField(blank=False, null=False, db_index=True)
```

Comment:
I don't know why this pull request is so funky, but this is the actual change made.